configuration/printers/prusa-mk3s/display.cfg

```diff
@@ -9,6 +9,14 @@ d7_pin: display_d7_pin
 encoder_pins: ^display_encoder_pin1,^display_encoder_pin2
 click_pin: ^!display_click_pin
 
+[output_pin LCD_backlight_pin]
```

Comment:
This section should be removed. RatOS currently has no concept of lcd displays. There's a feature in development to allow for "accessories" to support arbitrary things like this, until then it will have to be a commented out section in the printer.cfg template.